includes/Modules/Ads.php

```diff
@@ -139,7 +139,8 @@ protected function setup_assets() {
 				$assets[] = new Script(
 					'googlesitekit-ads-pax-integrator',
 					array(
-						'src'          => 'https://www.gstatic.com/pax/latest/pax_integrator.js',
+						// When updating, mirror the fixed version for google-pax-sdk in package.json.
+						'src'          => 'https://www.gstatic.com/pax/1.0.8/pax_integrator.js',
```

Comment:
Nice idea, but `package.json` doesn't exist in the built plugin. We could potentially copy it at build time into our `dist/config.php` though 🤔 

The version used here is really more important to be intentional about changing though. I think it might be too easy to change it unintentionally if we pulled it from package.json (which also prefers using the version constraint rather than a fixed version).